README.md

```diff
@@ -1,10 +1,23 @@
-## Dev
+# Dev
+
+This project uses the following frameworks:
+
+*EKSCTL* is used to manage the AWS EKS infrastructure. Creating the K8s cluster, setting the instance type, etc.
```

Comment:
probably worth adding a note that we are currently testing on AWS, but eventually intend to support other cloud providers as well.